Pipfile

```diff
@@ -6,14 +6,15 @@ name = "pypi"
 [packages]
 gitpython = "*"
 requests = ">=2.20.0"
+commonmark = "*"
 
 [dev-packages]
 pylint = "*"
 black = "*"
 pytest = "*"
 six = "*"
 codecov = "*"
-"flake8" = "*"
+flake8 = "*"
```

Comment:
We should ask @gkapfham to confirm that removing the `"` surrounding `flake8` here will have no negative consequences, just to be sure.